engine/execution/ingestion/engine.go

```diff
@@ -692,9 +702,15 @@ func (e *Engine) executeBlock(
 		e.executionDataPruner.NotifyFulfilledHeight(executableBlock.Height())
 	}
 
-	e.unit.Ctx()
-
 	e.stopControl.blockExecuted(executableBlock.Block.Header)
```

Comment:
we don't need to pass the block to stopControl anymore, right?

Reply:
That is correct, but I wanted to do that in a followup PR, so I don't put to much logic in one PR.

As far as I can see the engine wont need `e.stopControl` at all.